examples/config/networks/base.json

```diff
@@ -0,0 +1,21 @@
+{
+  "network_type": "EVM",
+  "slug": "base",
+  "name": "Base",
+  "rpc_urls": [
+    {
+      "type_": "rpc",
+      "url": {
+        "type": "plain",
+        "value": "https://mainnet.base.org"
+      },
+      "weight": 100
+    }
+  ],
+  "chain_id": 8453,
+  "block_time_ms": 2000,
+  "confirmation_blocks": 12,
+  "cron_schedule": "0 */1 * * * *",
+  "max_past_blocks": 18,
```

Comment:
With this configuration, I think `max_past_blocks` can be:

`(60000/2000) + 12 + 1 = 43`

Calculate as: (cron_interval_ms/block_time_ms) + confirmation_blocks + 1 (defaults to this calculation if not specified).
Example for 1-minute Ethereum cron: (60000/12000) + 12 + 1 = 18 blocks.